tests/lib/snaps/test-snapd-content-advanced-plug/meta/snap.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-snapd-content-advanced-plug
+summary: A more complex snap for testing content interface 
```

Comment:
Thanks for this test!

Reply:
More is coming here :) This is just a start